changelog.d/8767.bugfix

```diff
@@ -0,0 +1 @@
+Cap the version of `prometheus-client` in the docker image to match Synapse's requirements.
```

Comment:
shouldn't this say what bug is being fixed, not how we are doing so?

Reply:
You're right, I've updated it. Does it look any better?